### DISCUSSION TITLE
Find the 5 cases of 3-digit numbers that add up to 23

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#672] [Galaxy Trivia] [answered]
<img width="220" alt="image" src="https://user-images.githubusercontent.com/5120290/235726164-b31b314d-647a-4dc6-92b3-91281837fb5b.png">

Directions: ➡️⬇️⬅️⬆️

**Comment by peterbaccaro** [comment, marked as answer]
![23](https://user-images.githubusercontent.com/74666079/235749369-72943bcd-b376-4845-90b1-1ad3110dd187.png)
